README.md

```diff
@@ -32,10 +32,14 @@ See full example [here](https://github.com/web-eid/web-eid-spring-boot-example).
 
 See below for alternative installation options.
 
-### 2. Add authentication support
+### 2. Activate CSRF protection
+
+Web eID requires CSRF protection. Enable and configure CSRF protection in your web application and include the CSRF token in the POST requests in the code examples below.
+
+### 3. Add authentication support
 
 Copy the following code to the authentication page of your web application,
```

Comment:
Should probably be reworded.  
A website might have a different mechanism for loading 3rd party libraries, copy-paste is not always the right option.  
The code also contains plain-js event handling which is not desirable in a SPA framework.  
IMO, the code we provide should just be an example, not something to be copied directly.

Reply:
I agree that this can be misunderstood as normative. How to keep the wording minimal as the point here is to get a realist solution up and running in no time?